app/assets/stylesheets/modules/full_text_highlight.scss

```diff
@@ -0,0 +1,32 @@
+dt.blacklight-full_text_tesimv {
```

Comment:
We unfortunately have to provide the qualifying selector because the same class is on two elements that we want to treat differently.  This was pulled over from the previous PR, and seems like is safe to ignore in this case.